wordpress-mobile/WordPress-iOS
# Issue/5380 background refresh on reader

**Fixes** #5380 

**To test:**

**A) Test on the Simulator**

Regular Background Fetch
1. Start the app from XCode
2. Go to Reader->AnyFeed (I suggest a8c feed since there's always sth happening there)
3. Simulate Home button
4. XCode->Debug->Simulate Background Fetch

App Started from a Background Fetch
1. Edit Run Scheme, select Background Fetch: Launch App due to a background fetch event

On both cases you can check that the refresh was performed by looking at the console, and of course if new content was fetched it should be present on the reader.

**B) Test on the Device**

You can perform the aforementioned tests on the device, but is not too much fun. What is great is when you go into the app and the content is updated :)

1. Deploy the app to your device and go to one of your reader feeds.
2. Press home and continue with your life.
3. After a few hours go back to the app and check the feed, it should be beautifully updated.

Info on background refresh can be found on https://developer.apple.com/videos/play/wwdc2013/204/ minute 8 to 22

Needs review: @aerych
